src/views/LotteryV2/components/BuyTicketsModal/generateTicketNumbers.tsx

```diff
@@ -0,0 +1,50 @@
+import { LotteryTicket } from 'config/constants/types'
+
+/**
+ * Return a random number between (and inclusive of) minNumber & maxNumber.
+ */
+function generateRandomNumber(minNumber: number, maxNumber: number): number {
+  const minInt = Math.ceil(minNumber)
+  const maxInt = Math.floor(maxNumber)
+  return Math.floor(Math.random() * (maxInt - minInt + 1) + minInt) // The maximum is inclusive and the minimum is inclusive
+}
```

Comment:
https://lodash.com/docs/4.17.15#random

Reply:
Well damn ain't that a lot neater. Updated